gitlab/objects.py

```diff
@@ -520,6 +520,21 @@ class ProjectCommitManager(BaseManager):
     obj_cls = ProjectCommit
 
 
+class ProjectCommitStatus(GitlabObject):
+    _url = '/projects/%(project_id)s/statuses/%(commit_id)s'
+    canUpdate = False
+    canDelete = False
+    requiredUrlAttrs = ['project_id', 'commit_id']
+    requiredCreateAttrs = ['state']
+    optionalCreateAttrs = ['description', 'name', 'ref', 'target_url']
+    requiredGetAttrs = []
+    requiredUpdateAttrs = []
+    requiredDeleteAttrs = []
+
```

Comment:
Add a blank line to make the pep8 tests pass (you can run `tox -epep8` to check)